src/controllers/version/metadata.rs

```diff
@@ -43,20 +43,15 @@ pub fn authors(req: &mut dyn Request) -> AppResult<Response> {
         .order(version_authors::name)
         .load(&*conn)?;
 
-    // It was imagined that we wold associate authors with users.
-    // This was never implemented. This complicated return struct
-    // is all that is left, hear for backwards compatibility.
     #[derive(Serialize)]
     struct R {
-        users: Vec<EncodablePublicUser>,
         meta: Meta,
     }
     #[derive(Serialize)]
     struct Meta {
         names: Vec<String>,
     }
     Ok(req.json(&R {
-        users: vec![],
```

Comment:
@JohnTitor we discussed this in the team meeting yesterday. we would prefer for this to stay in the API response for now to avoid potentially breaking changes. the database changes are fine, but it would be great it you could remove this API change from the PR :)